test/Interop/Cxx/class/type-classification-loadable-silgen.swift

```diff
@@ -52,21 +52,6 @@ func pass(s: StructWithSubobjectDefaultedCopyConstructor) {
   // CHECK: bb0(%0 : $StructWithSubobjectDefaultedCopyConstructor):
 }
 
-// CHECK-LABEL: sil hidden [ossa] @$s4main4pass{{.*[ (]}}StructWithMoveConstructor)
```

Comment:
As per the comment on L3331 and the semi-related discussion in #34649, we should not import decls that we cannot destroy or copy. These are all decls which we cannot copy, so we should never have been able to import them. This was a bug (see comment above).